test/e2e/apps/statefulset.go

```diff
@@ -125,7 +125,7 @@ var _ = SIGDescribe("StatefulSet", func() {
 
 		// This can't be Conformance yet because it depends on a default
 		// StorageClass and a dynamic provisioner.
-		ginkgo.It("should provide basic identity", func() {
+		ginkgo.It("should provide basic identity [feature: CloudProvider]", func() {
 			ginkgo.By("Creating statefulset " + ssName + " in namespace " + ns)
 			e2epv.SkipIfNoDefaultStorageClass(c)
```

Comment:
Now that I think about it some more, many of these tests skip if there's no default storageclass installed. What if instead of adding the feature tag to this test, we remove the default storageclass from kube-up deployments?

The main downside of adding the feature tag is downstream CI jobs need to update all of their configs to add a focus on this new tag, which also conflicts with the blanket skip regex on [Feature:*]

Reply:
turned off default storageclass in #104722 and observing what tests fail to skip